doc/content/getting-started/aws/ecs/deployment/_index.md

```diff
@@ -121,6 +121,13 @@ For deployments that connect to Packet Broker, you need to configure your Packet
 
 For multi-tenant deployments that use tenant billing through Stripe (see the [Stripe Billing reference]({{< ref "/reference/stripe" >}})), you need to configure the Stripe API key (starting with `sk_`) and Stripe endpoint secret key (starting with `whsec_`). It is possible to add or update this in the future.
 
+For the `Gateway Secrets Encryption Key Value` parameter {{< new-in-version "3.10" >}}, provide anAES-128 Key in Base64. The following command can be used to randomly generate one;
```

Comment:
- `anAES` is missing a space
- `;` should be a `:`

Reply:
😞